docs/source/projects/architecture/content-architecture.rst

```diff
@@ -135,3 +136,41 @@ IPython also includes a parallel computing framework,
 `IPython.parallel <https://ipyparallel.readthedocs.io/en/latest/>`_. This
 allows you to control many individual engines, which are an extended version
 of the IPython kernel described above.
+
+
+JupyterHub and Binder
+=====================
+
+JupyterHub is a multi-user Hub that spawns, manages, and proxies multiple instances of the
+single-user Jupyter notebook server. This can be used to serve a variety of interfaces
+and environments, and can be run on many kinds of infrastructure. JupyterHub on Kubernetes
+is a Helm Chart for running JupyterHub on kubernetes infrastructure, and BinderHub is a
+customized JupyterHub deployment for sharable, reproducible interactive computing environments.
+
+The links below describe the architecture of JupyterHub and several distributions of
+JupyterHub.
+
+* :doc:`JupyterHub core architecture <hub:reference/technical-overview>`
+* :doc:`JupyterHub for Kubernetes architecture <z2jh:administrator/architecture>`
+* :ref:`BinderHub architecture <bhub:diagram>`
+
+
+JupyterLab
+==========
+
+JupyterLab is a flexible, extensible interface for interactive computing. Below
+are a few links that are useful for understanding the JupyterLab architecture.
+
+* :doc:`JupyterLab document model <lab:developer/documents>`
+* :doc:`JupyterLab notebook model <lab:developer/notebook>`
+* :doc:`Design patterns in JupyterLab <lab:developer/patterns>`
+
+Projects overview
```

Comment:
Maybe it would make more sense to have the visual overview at the very beginning of the section? What do you think?

Reply:
I agree in theory - though the reason I put it at the bottom was because it's a bit out-of-date and so doesn't quite match the content of the page itself. Not quite sure what to do about that but I didn't have the bandwidth to update this figure :-/

Reply:
Then, I believe we should leave it like this until we update the figure. Definitely not block the PR because of it.

I'll happily work on updating it in the following days and move it at the begging of the section once we're happy with it. What do you think?

Reply:
sounds great - let me know if I can help you update the figure as well, happy to do a little virtual whiteboarding session or something

Reply:
I'll add a "TODO" so it's clear what should be done